eqcorrscan/utils/catalog_to_dd.py

```diff
@@ -11,7 +11,8 @@
 import numpy as np
 import logging
 from collections import namedtuple, defaultdict, Counter
-from functools import partial
+from obspy.core import stream
+from orderedset import OrderedSet
```

Comment:
This seems to be causing the tests to fail - `orderedset` doesn't exist - is this another dependency that needs to be added? Is there a way to not add this dependency and just run `list(set(thing))` to ensure order is retained?

Reply:
Right, I noted that when I first implemented it and indeed adding another dependency is not optimal. Found it a bit difficult to get my head around a good solution without the `OrderedSets` that wasn't several lines long, but now I realized it works with just one extra line of code.